src/config/index.ts

```diff
@@ -497,3 +497,25 @@ export const TRUST_ENTRY_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_WITH_PARA
 export const TRUST_INFO_WITH_PARAMS_URL = REGISTER_AN_OVERSEAS_ENTITY_WITH_PARAMS_URL + TRUST_INFO_PAGE;
 export const TRANSACTION_ID_URL_KEY = "/transaction/";
 export const SUBMISSION_ID_URL_KEY = "/submission/";
+
+// Check-your-answers page links
+export const ENTITY_CHANGE_NAME_WITH_PARAMS = OVERSEAS_NAME_WITH_PARAMS_URL + ENTITY_NAME;
+export const PRESENTER_CHANGE_FULL_NAME_WITH_PARAMS = PRESENTER_WITH_PARAMS_URL + "#full_name";
+export const PRESENTER_CHANGE_EMAIL_WITH_PARAMS = PRESENTER_WITH_PARAMS_URL + "#email";
```

Comment:
You should use the `EMAIL` constant already defined here.

Ideally, we'd have a new `FULL_NAME` constant too for "#full_name".

Reply:
Good spot Martin. Have done so and pushed. Cheers.